lgc/patch/PatchResourceCollect.cpp

```diff
@@ -3059,43 +3048,39 @@ void PatchResourceCollect::updateOutputLocInfoMapWithPack() {
         origLocInfo.setStreamId(streamId);
       }
       const unsigned origLocation = origLocInfo.getLocation();
-      bool isUsed = xfbOutputLocs[streamId].count(origLocation) > 0 || flatInputLocs.count(origLocation) > 0 ||
-                    customInputLocs.count(origLocation) > 0 || nextStageInputLocInfoMap.count(origLocInfo) > 0;
-      if (!isUsed)
+      const bool hasNoMappedInput = (nextStageInputLocInfoMap.find(origLocInfo) == nextStageInputLocInfoMap.end());
+      if (hasNoMappedInput && xfbOutputLocs[streamId].count(origLocation) == 0)
         m_deadCalls.push_back(call);
+      else if (hasNoMappedInput)
+        noMappedCalls.push_back(call);
```

Comment:
Might be cleaner as:
```suggestion
      const bool hasNoMappedInput = (nextStageInputLocInfoMap.find(origLocInfo) == nextStageInputLocInfoMap.end());
      if (hasNoMappedInput) {
        if (xfbOutputLocs[streamId].count(origLocation) == 0)
          m_deadCalls.push_back(call);
        else
          noMappedCalls.push_back(call);
      }
```

Reply:
Thanks! Fixed.